axon-server-connector/src/main/java/org/axonframework/axonserver/connector/command/AxonServerRemoteCommandHandlingException.java

```diff
@@ -22,6 +22,7 @@
 
 /**
  * Exception indicating a problem that was reported by the remote end of a connection.
+ * By default, stack trace is not generated.
```

Comment:
I think the wording is slightly off. Either "a" should be added before "stack trace", or stack trace should be plural:

```suggestion
 * By default, stack traces are not generated.
```

Reply:
I see you've added this text to every exception. Instead of commenting on every file, I'm using this comment as a pointer that I would recommend this change for every file.

Reply:
How about something like this?

```
An AxonServer Exception which is thrown on a Query Handling exception.
<p/>
By default, a stack trace is not generated for this exception. However, the stack trace creation can be enforced explicitly 
via the constructor accepting the writableStackTrace parameter.
```

In addition, I'll remove the"`this is a default`" note from the `writeableStackTrace` parameter description. WDYT?

Reply:
It seems fine to me, as long as the first sentence correctly shares whether it is a command, query, and/or remote exception. I don't have hard feelings about adding Axon Server in that sentence, yes or no.